travis/setup_conda_windows.sh

```diff
@@ -6,12 +6,14 @@ if [[ $DEBUG == True ]]; then
 fi
 
 if [[ -z "${MINICONDA_VERSION}" ]]; then
-    MINICONDA_VERSION=4.5.4
+    MINICONDA_VERSION=4.6.14
```

Comment:
is 4.7 not working with windows?

Reply:
It wasn't available on chocolatey a 1.5 months ago, but does appear to be now: https://chocolatey.org/packages/miniconda3